### PR TITLE
[Merged by Bors] - doc(SetTheory/Game/PGame): expand docstrings for `≤` and `⧏`

### PR DESCRIPTION
**Pull request by vihdzp**
---

The docstring doesn't reflect how the relations are defined, but I think it's the best way to think about them in practice. 

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by urkud** [comment]
Could you please also explain the notation `{ _ | _ }` somewhere in the docs? The first time it appears in the docs is the explanation of the definition of addition of `PGame`s.

File: Mathlib/SetTheory/Game/PGame.lean [hunk 1]
```diff
@@ -349,15 +349,15 @@ instance isEmpty_one_rightMoves : IsEmpty (RightMoves 1) :=
 
 /-- The less or equal relation on pre-games.
 
-If `0 ≤ x`, then Left can win `x` as the second player. -/
+If `0 ≤ x`, then Left can win `x` as the second player. `x ≤ y` means that `0 ≤ y - x`. -/
```
Comment (b-mehta) [inline Mathlib/SetTheory/Game/PGame.lean:352]: You mention this isn't how it's actually defined - is there a lemma saying these are equivalent though? If so, let's reference it here.

Comment (vihdzp) [reply Mathlib/SetTheory/Game/PGame.lean]: I've added that to the docstring.

**Comment by vihdzp** [comment]
The notation `{ _ | _ }` isn't actually defined in Mathlib, it's just meant to be the standard notation for games, i.e. `{ Lᵢ | Rⱼ }` is the game with `Lᵢ` as left options are `Rⱼ` as right options.

File: Mathlib/SetTheory/Game/PGame.lean [hunk 2]
```diff
@@ -349,15 +349,17 @@ instance isEmpty_one_rightMoves : IsEmpty (RightMoves 1) :=
...
 instance le : LE PGame :=
   ⟨Sym2.GameAdd.fix wf_isOption fun x y le =>
       (∀ i, ¬le y (x.moveLeft i) (Sym2.GameAdd.snd_fst <| IsOption.moveLeft i)) ∧
         ∀ j, ¬le (y.moveRight j) x (Sym2.GameAdd.fst_snd <| IsOption.moveRight j)⟩
 
 /-- The less or fuzzy relation on pre-games.
 
-If `0 ⧏ x`, then Left can win `x` as the first player. -/
+If `0 ⧏ x`, then Left can win `x` as the first player. `x ⧏ y` means that `0 ⧏ y - x`.
+See `PGame.lf_iff_sub_zero_lf`.  -/
```
Comment (Command-Master) [inline Mathlib/SetTheory/Game/PGame.lean:362]: Perhaps add that it's defined as `¬y ≤ x`? I don't think that's complex to explain

**Comment by urkud** [comment]
> The notation `{ _ | _ }` isn't actually defined in Mathlib, it's just meant to be the standard notation for games, i.e. `{ Lᵢ | Rⱼ }` is the game with `Lᵢ` as left options are `Rⱼ` as right options.

I think that it should be explained anyway (and the explanation should tell that this notation is not used in Mathlib).

File: Mathlib/SetTheory/Game/PGame.lean (hunk 2 above)
Comment (vihdzp) [reply Mathlib/SetTheory/Game/PGame.lean]: That seems useful, I've added it to the docstring.

**Comment by vihdzp** [comment]
I've added the remark on `{xL | xR}` to the module docstring.

**Review by Command-Master — APPROVED** [review]

**Comment by urkud** [comment]
Thanks! :tada:
bors merge

**Pull request closed without merging**